docs/08_crosscutting_concepts/08_crosscutting_concepts.md

```diff
@@ -2,4 +2,212 @@
 title: Crosscutting concepts
 ---
 
-# Crosscutting concepts
\ No newline at end of file
+# Crosscutting concepts
+
+
+# Deployment view
+
+## Frontend
+
+### 8.1 Dependencies
+For the UI components:<br>
+https://ant.design/components/overview/
+
+For routing: <br>
+https://reactrouter.com/
+
+I18N Internationalization: <br>
+https://react.i18next.com/
+
+MSAL User authentication: <br>
+https://github.com/microsoft/Federal-App-Innovation-Community/tree/main/topics/modern-auth/React-MSAL-AAD
+
+
+### 8.3 User Interface
+The design was previously mapped in Figma to give the developer a clear idea of the final product and to create a common basis for discussion.
+
+Link to Figma Design: <br>
+https://www.figma.com/file/umOKw8yF0RbQyPkU4B7bIe/SmashGrade?type=design&node-id=111-2817&mode=design&t=JDM3bRnav0m3z2Re-0
+
+
+As a UI library we use Antd, which allows us to use predefined base components such as forms, buttons, etc...
+
+#### Global SCSS
+In React we can define a global scss file to apply some styles to all pages, such as the font-color or text-size.
+
+Example:
+```scss
+body {
+  font-family: 'Roboto', sans-serif;
+}
+```
+
+#### Colors Module SCSS
+So that colors for backgrounds, buttons and validators have the same colors on all pages, a Colors module is also created in which the standard colors can be recorded and reused.
+
+Example:
+```scss
+$colorPrimary: #c6264e;
+$colorSecondary: #d9d9d9;
+$colorTertiary: #ebebeb;
+$colorTextHeading: #ffffff;
+$colorBgContainer: #ffffff;
+$colorBgContainerHighlight: lightgrey;
+$colorLink: #c6264e;
+$colorLinkHover: #3d3d3c;
+$colorLinkActive: #95223f;
+$colorRatingGood: #b1ffa5;
+$colorRatingBad: #ff728b;
+$colorRatingMedian: #ffd76f;
+
+:export {
+    colorPrimary: $colorPrimary;
+    colorSecondary: $colorSecondary;
+    colorTertiary: $colorTertiary;
+    colorTextHeading: $colorTextHeading;
+    colorBgContainer: $colorBgContainer;
+    colorBgContainerHighlight: $colorBgContainerHighlight;
+    colorLink: $colorLink;
+    colorLinkHover: $colorLinkHover;
+    colorLinkActive: $colorLinkActive;
+    colorRatingGood: $colorRatingGood;
+    colorRatingBad: $colorRatingBad;
+    colorRatingMedian: $colorRatingMedian;
+}
+```
+
+#### Color Schema
+We developed the color scheme according to the HFTM corporate design.
+
+| Farbvariable   | Hex-Wert  |
+| -------------- | --------- | 
+| Primary        | `#C6264E` | 
```

Comment:
If you wanted you could do something like this to actually show the colours: ```| <span style="color:#C6264E">Primary</span>| `#C6264E` | ```

Reply:
adapted as proposed